addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,223 +1,265 @@
-$gold: #ffcc00;
+$gold: #eca801;
 $silver: #cccccc;
 $bronze: #eea91e;
 
 .website_forum {
+    .o_forum_ask_btn {
+        @include media-breakpoint-up(md) {
+            box-shadow: $box-shadow;
+            width: 200px;
+        }
+    }
+
+    .o_wforum_nav.nav.nav-pills .nav-link {
+        line-height: 1;
+        @include o-hover-text-color($body-color, $link-color);
+
+        .fa {
+            opacity: 0.5;
+        }
 
-    .question-block .question-name {
-        max-width: 80%;
+        &:hover, &.active {
+            .fa {
+                opacity: 1;
+            }
+        }
+
+        &.active {
+            background-color: rgba(theme-color('info'), 0.1);
+            color: darken(theme-color('info'), 15%)!important;
+        }
     }
 
-    .o_post_content {
+    // Single Post
+    .o_wforum_post, .note-editable.card-block {
+        word-wrap: break-word;
+
         pre {
             padding: 1.5rem;
             border-radius: 5px;
             color: $white;
             background-color: $gray-700;
         }
+
         blockquote {
             position: relative;
-            padding-left: 3rem;
-            border-left: 2px solid $gray-500;
-            font-style: italic;
-            font-weight: lighter;
+            padding-left: 1em;
+            border-left: .25em solid $gray-500;
+            color: $gray-500;
         }
-        word-wrap: break-word;
     }
 
-    .vote {
-        min-width: 38px;
-        margin-right: 12px;
-        .vote_count {
-            font-weight: bold;
-            line-height: 12px;
-            font-size: 160%;
-        }
-        .vote_down,
-        .vote_up {
-            font-size: 130%;
-        }
-    }
+    .o_wforum_readable {
+        max-width: 700px;
 
-    .post_to_validate {
-        overflow: hidden;
-        .validate_answer {
+        > p:empty:first-child:last-child {
             display: none;
         }
-    }
 
-    .validate_answer {
-        @include o-position-absolute(0, auto, 0, 0);
-        width: 4px;
-        height: 100%;
-        z-index: 1; // On top of card content if an alert is displayed inside
+        p {
+            margin-bottom: 0.5rem;
 
-        > .bg-success {
-            @include o-position-absolute(0, 0, 0, 0);
+            > br:first-child:last-child {
+                display: none;
+            }
         }
-        > span {
-            @include o-position-absolute(10px, 50%);
-            transform: translateX(50%);
-            font-size: 20px;
+    }
+
+    textarea.load_editor {
+        opacity: .5;
+        background-color: gray('300');
+        min-height: 100px;
```

Comment:
I'll explain you what I'm doing, tell me if it's fine.

Initially I wanted to add a class to $target elements  in `wysiwyg.js`, but  there is no way to know if an element will be replaced until the rendering start, and it's too late at that point.

I may simply design a CSS a class, something like `.o_wysiwyg_loader`  in `wysiwyg.scss` and use this class in both forum and profile, apparently the only two frontend modules that need this.

Is it ok?

Reply:
> Same for the o_wforum_handler... :)

I can't find this class

Reply:
> could we not make this generic in website instead of defining it in website_forum?

https://github.com/odoo/odoo/pull/35749/commits/5ae634b90ba0a648772d3258c221b1a0cbeeee97

Reply:
> I can't find this class

Typo of course I meant `o_wforum_loader`

Your commit follows the idea but:
- Define the style and classes in web_editor, not website
- Don't use `calc`, you can use a translate transform to move it up 50%.
- Don't use difficult-to-read prone-to-errors scss to hide the loading icon, remove it in JS like you did before
- The loading icon should not be xml to add by yourself, it should be added in JS too

Also:
- Ideally, remove the `load_editor` class, use `o_wysiwyg_loading` instead and define a publicWidget (old animations) in web_editor assets frontend to enable all editors and thus share the code for loading them (not duplicate in forum and profile and other places later).

I can do the last point once I can if you want :wink:

Reply:
> The loading icon should not be xml to add by yourself, it should be added in JS too

> Don't use difficult-to-read prone-to-errors scss to hide the loading icon, remove it in JS like you did before

Yep, but it's the point... when the wysiwyg starts it's already too late to render a loader.
I have the browser cache disabled and it takes at least 1sec. 

> I can do the last point once I can if you want 😉

Yep, I think it's better... remember that the target `textarea` and `o_wforum_loader ` must be wrapped in a `position-relative` div 😉

Reply:
Mmh, I forgot about the lazy loading... I'll have to think about it :/

Reply:
Added a commit on top to solve all of this.